docs/docs/docs/tutorial/actions.md

```diff
@@ -79,4 +80,4 @@ Now we are ready to test drive our changes. Click the button and you should see
 
 - Add another custom action which transforms the input value to Uppercase. You may override existing properties on the input or create a new one
 
-**Want to dive deeper?** - [Go in depth](../in-depth/06_actions.html), or move on with the tutorial
+**Want to dive deeper?** - [Go in depth](../in%20depth/actions.html), or move on with the tutorial
```

Comment:
Links in md files must be working on github. It means it should point to another .md file and converted on .html on build time.
Previous generator allowed it.